### PR TITLE
Update Java version

### LINKED ISSUE #269: Update Web3Signer build requirements to Java 21
Update to mention use of Java 21

Related to: https://github.com/Consensys/web3signer/pull/995

### PR DESCRIPTION
**Pull request by joaniekube**
Fixes #269

### CODE REVIEW

**Comment by joaniefromtheblock** [comment]
@usmansaleem please review when you have a chance. Thanks a million

**Review by usmansaleem — APPROVED** [review]
LGTM

File: docs/get-started/build-from-source.md [hunk 1]
```diff
@@ -8,11 +8,11 @@ sidebar_position: 3
 
 ## Prerequisites
 
-- [Java JDK](https://www.oracle.com/java/technologies/javase-downloads.html)
+- [Java JDK](https://jdk.java.net/21/)
```
Comment (m4sterbunny) [inline docs/get-started/build-from-source.md:11]:
```suggestion
- [Java JDK](https://jdk.java.net)
```

Comment (m4sterbunny) [reply docs/get-started/build-from-source.md:11]: link goes to "superseded" notice

File: docs/get-started/install-binaries.md [hunk 2]
```diff
@@ -11,11 +11,11 @@ import TabItem from '@theme/TabItem';
 
 ## Prerequisites
 
-- [Java JDK](https://www.oracle.com/java/technologies/javase-downloads.html)
+- [Java JDK](https://jdk.java.net/21/)
```
Comment (m4sterbunny) [inline docs/get-started/install-binaries.md:14]: ditto above

**Review by m4sterbunny — APPROVED** [review]
LGTM

**Pull request merged**